### PR TITLE
Limit the size of content on very large screens

### PR DESCRIPTION
**Pull request by didaquis**
I've added some media queries to limit the size of content on very large screens. 

I'm not a CSS expert so I would like to get feedback from other users about these changes.

### CODE REVIEW

**Comment by didaquis** [comment]
@zearin would you like to give me some feedback?

File: views/layout.swig [hunk 1]
```diff
@@ -68,50 +68,55 @@
...
-        <br>
-        This website is <a href="https://github.com/jsdoc3/jsdoc3.github.com">open source</a> and is
-        licensed under the <a rel="license" href="http://creativecommons.org/licenses/by-sa/3.0/">
-        Creative Commons Attribution-ShareAlike 3.0 Unported License</a>.
+        <div class="content-block">
+            <a class="license-badge" href="http://creativecommons.org/licenses/by-sa/3.0/">
+            <img alt="Creative Commons License" class="license-badge" src="images/cc-by-sa.svg"
+                width="80" height="15" /></a>
+            <br>
+            Copyright &#169; 2011-2019 the
```
Comment (Zearin) [inline views/layout.swig]: © 2011–**2020**, now ;-)

**Comment by Zearin** [comment]
(Sorry it took me so long to respond!)

Looks okay to me.  If there are further tweaks, they can happen in future Pull Requests.  If you’ve tested it on screens that big (which I don’t have at home, so I’ll just trust you), I say it’s worth a merge.

**Comment by didaquis** [comment]
> If you’ve tested it on screens that big (which I don’t have at home, so I’ll just trust you), I say it’s worth a merge.

Yes I tried on a double monitor (extended desktop).

File: views/layout.swig (hunk 1 above)
Comment (didaquis) [reply views/layout.swig]: I'm not sure about that. On my opinion that change will be another topic (pull request) completely different. 😉

**Comment by didaquis** [comment]
@hegemonic ping!  ;-)

**Pull request closed without merging**